appendix_meta_rules.md

```diff
@@ -536,29 +477,36 @@ To keep the file names interoperable use the following:
 
 As a best practice use the prefix `mr_filter_`
 
+<!-- TODO: finalize the rx schema-->
 ### Schema
 
 ```yaml
 title: //str
 description: //str
 action: //str
 type: //str
-rules: //map
-logsource:
+logsource: # optional
     product: //str
     service: //str
-selection:
+global_filter:
     type: //rec
+    required:
+      detection: //rec
+    optional:
+      <selections> // rec
+      rules: //arr # no defined rules means matching on all rules
+
 ```
 
 ### Syntax
 
-Like Sigma rules, "Filter" rules have a title.  
+Like Sigma rules, "Filter" rules have a title.
 They don't have an id or level as they use the one from the referenced rules.
+<!-- I think they should have a id, even if we don't use it (yet) -->
```

Comment:
```suggestion
Like Sigma rules, "Filter" rules have a `title` and an `id` to facilitate their management.
It has no other meta data like level or status because its purpose is to complete a sigma rule.

```

Reply:
I updated the suggestion from "complete" to "enrich an existing" Sigma rule. I hope you agree @frack113 ?